### DISCUSSION TITLE
How to create another project in the local instance of Supabase?

### DISCUSSION THREAD

**Discussion by illright** [supabase/supabase#5759] [Questions]
When I run a local instance of Supabase with the Supabase CLI, the studio presents me with a Default Organization and a Default Project. The dashboard looks identical to the app.supabase.com version, however, which leads me to believe that there would similarly be an option to create another project.

If there is no such option, could this be explicitly written in the Dashboard of the local Studio? And if possible, I'd like to be able to rename the "Default" names.

**Comment by GaryAustin1** [comment]
From kiwicopple awhile back answering ???:

    ??? Can't even seem to be able to create another project or delete the default project?

This is by design: https://github.com/supabase/supabase/tree/master/studio#whats-included
It is not intended for managing the deployment and administration of projects - that's out of scope.

The Studio will not offer [creating/administrating projects](https://github.com/supabase/supabase/tree/master/studio#managing-project-settings). The rough rule of thumb is....
https://github.com/supabase/supabase/issues/4440

**Reply by illright** [reply]
Is it possible to allow customizing the project name/team name through the Supabase CLI? Also I would appreciate if this information about not supporting several projects was presented in the CLI or in the Studio interface

**Reply by pessimo** [reply]
for studio, I understand it's by design. but is there any method for local development to create a custom project?

**Reply by saltcod** [reply]
@kiwicopple there was a PR just this week I thought that addressed renaming the default project? I can't find it, but I'm almost certain I saw it yesterday or the day before. Do you recall?
